src/accounts/models.py

```diff
@@ -42,6 +42,10 @@ class User(BasicUser):
     activation_key=models.CharField(_('activation key'), max_length=40, editable=False)
     user_text=models.CharField(null=True, blank=True, max_length=500, help_text = _("Custom text which will be shown to this student."))
     accepted_disclaimer=models.BooleanField(default=False, help_text="Whether the user accepted the disclaimer.")
+    #'attestation_emails' to check whether the user wishes to receive an email notification for each attestation.
+    attestation_emails = models.BooleanField(default=True, help_text="recive confirmation Email for each task has been attested.")
+    #'upload_confirm_emails' to check whether the user wishes to receive an email notification for each upload they make.
+    upload_confirm_emails = models.BooleanField(default=False, help_text="recive confirmation Email for each file has been uploaded.")
```

Comment:
see above

Reply:
As you changed the default to false, this is a breaking change that should be reflected in the pull request message and maybe also in the changelog (if this exists for this project)

Reply:
I guess there is no changelog. So putting it prominent in the PR description is sufficient.